cmd/spec/path.yaml

```diff
@@ -4,7 +4,7 @@ info:
   license:
     name: GPL-3.0
   title: provisioning-api
-  version: 0.0.2
+  version: 1.0.0
```

Comment:
Are we sure we want to bump directly to 1.0? I'd wait with that for the time we go live (to production), no?

Reply:
I agree that 1.0.0 is the best start, let's just not touch the version until our official release and only then let's start bumping the number. No guarantees for now.

Reply:
The main idea is to start already with 1.0.0, and start bumping it only after official deployment
BTW, the provisioning API is going to be consumed by other services in stage env, so it makes sense to me to start versioning when our API is been used.

Reply:
We never get to the v1 state, I am a big advocate of releasing 1.0.0 as soon as possible.

What is important is the contract which is currently: "no guarantees, this is a beta". And I think it is fair to say that. Maybe if OpenSPEC supports a "beta" flag let's set it.

Reply:
I haven't found any beta flag in the spec, so the description workaround seems fine to me